src/CheckBox/index.jsx

```diff
@@ -1,69 +1,62 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-import { inputProps } from '../asInput';
+import asInput from '../asInput';
 import newId from '../utils/newId';
 
-class CheckBox extends React.Component {
+class Check extends React.Component {
   constructor(props) {
     super(props);
 
-    this.handleClick = this.handleClick.bind(this);
-    if (this.props.onChange) {
-      this.onChange = this.props.onChange.bind(this);
-    }
+    this.onChange = this.onChange.bind(this);
 
     const id = newId('checkbox');
-    if (props.checked === 'true') {
-      this.state = {
-        id,
-        checked: true,
-      };
-    } else {
-      this.state = {
-        id,
-        checked: false,
-      };
-    }
+    this.state = {
+      id,
```

Comment:
is this `id` being used anywhere? if not, you can remove it (and the `const id` above, and the `newId` import)